mappings/net/minecraft/network/packet/s2c/play/UnlockRecipesS2CPacket.mapping

```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/class_2713 net/minecraft/network/packet/s2c/play/UnlockRecip
 	FIELD field_12408 action Lnet/minecraft/class_2713$class_2714;
 	FIELD field_12409 recipeIdsToInit Ljava/util/List;
 	FIELD field_12414 recipeIdsToChange Ljava/util/List;
+	FIELD field_25797 furnaceFilteringCraftable Lnet/minecraft/class_5411;
```

Comment:
```suggestion
	FIELD field_25797 options Lnet/minecraft/class_5411;
```
please rename the getter as well

Reply:
Fixed in 19ea7d3d586be3cecd60f0f8dd730aca5d634efc.